qemu/tests/cfg/block_io_with_unaligned_offset.cfg

```diff
@@ -0,0 +1,12 @@
+- block_io_with_unaligned_offset:
+    type = block_io_with_unaligned_offset
+    start_vm = no
+    img_file = /var/tmp/io_4kdisk.img
+    loop_dev = /var/tmp/loop_dev
+    content_file =/var/tmp/io_4kdisk
+    io_cmd = qemu-io -f raw -t none -c "writev -P 42 %d `cat ${content_file}`" %s
+    loop_cmd = losetup -f --show --sector-size 4096 --direct-io=on ${img_file}
+    pre_command = which qemu-io && truncate -s 1G ${img_file} &&
+    pre_command += ${loop_cmd} > ${loop_dev} &&
+    pre_command += yes 512 | head -n 1024 | tr '\n' ' ' >  ${content_file}
+    post_command = losetup -d `cat  ${loop_dev}`;rm -rf ${img_file}
```

Comment:
`cat ${loop_dev}` 

Do we want to use ' ' for this?
Other parts are ok for me.

Reply:
It is shell running symbol